docs/sources/operations/multi-tenancy.md

```diff
@@ -17,3 +17,19 @@ To run in multi-tenant mode, Loki should be started with `auth_enabled: true`.
 Loki can be run in "single-tenant" mode where the `X-Scope-OrgID` header is not
 required. In single-tenant mode, the tenant ID defaults to `fake`.
 
+## Multi-tenant Queries
+
+If run in multi-tenant mode, queries across different tenants can be enabled via
+`multi_tenant_queries_enabled: true` option in the querier. Once enabled multiple
```

Comment:
I'm trying for better wording than "queries across different tenants." What do you think?

```suggestion
If run in multi-tenant mode, queries that gather results from multiple tenants can be enabled with
the `multi_tenant_queries_enabled: true` configuration option in the querier. Once enabled, multiple
```

Reply:
That is much cleaner. Thanks. I'm wondering though if the results a from or for multiple tenants but that would be nit picking :slightly_smiling_face: